.travis.yml

```diff
@@ -171,7 +170,6 @@ script:
 
   # random python tests TODO(ekl): these should be moved to bazel
   - if [ $RAY_CI_PYTHON_AFFECTED == "1" ]; then RAY_FORCE_DIRECT=0 python -m pytest -v --durations=5 --timeout=300 python/ray/experimental/test/async_test.py; fi
```

Comment:
We are going to remove this tests soon (when we refactor the async plasma feature)